packages/__docs__/src/Methods/index.tsx

```diff
@@ -23,17 +23,15 @@
  */
 
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
 
 import { Table } from '@instructure/ui-table'
 
 import { compileMarkdown } from '../compileMarkdown'
-
-class Methods extends Component {
-  static propTypes = {
-    methods: PropTypes.array.isRequired
-  }
-
+import type { MethodsProps } from './props'
+import { propTypes, allowedProps } from './props'
+class Methods extends Component<MethodsProps> {
```

Comment:
Please leave an empty line above the class declaration for better readability :)